cleaner.py

```diff
@@ -43,15 +43,15 @@
 office_names = [
     # federal
     'President', 'Senate', 'House',
-    
+
     # statewide
     'Governor', 'Lieutenant Governor', 'Attorney General',
     'Secretary of State', 'State Treasurer',
     'State Superintendent of Public Instruction',
-    
+
     # state representatives
     'State Senate', 'State Assembly',
-    
```

Comment:
There should be no changes to whitespace here (3 lines).

Reply:
Oh, I think my editor automatically culled those trailing spaces.

Is it that the whitespace change isn't necessary? Or will removing the trailing spaces have an impact on how the code runs, or violate a style convention?

Reply:
Looks like the indents were removed. It's a style convention, shouldn't impact how the code runs. Blank lines should have the same indent as their code block. I think it makes editing easier. The main point is there should not be changes to lines that are not part of the commit.

Reply:
There's plenty of debate on whether blank lines should have indentation in Python. For me, when invisible chars are displayed in the editor, using the indents looks more consistent.
https://stackoverflow.com/questions/2727988/python-indentation-in-empty-lines

Reply:
Thanks for the fix.